### PR TITLE
Update help for "short-full" and shorten to 80 columns

### PR DESCRIPTION
**Pull request by keszybz**
Follow up for #3880.

### CODE REVIEW

File: src/journal/journalctl.c [hunk 1]
```diff
@@ -297,9 +297,9 @@ static void help(void) {
                "  -n --lines[=INTEGER]     Number of journal entries to show\n"
                "     --no-tail             Show all lines, even in follow mode\n"
                "  -r --reverse             Show the newest entries first\n"
-               "  -o --output=STRING       Change journal output mode (short, short-iso,\n"
-               "                                   short-precise, short-monotonic, verbose,\n"
-               "                                   export, json, json-pretty, json-sse, cat)\n"
+               "  -o --output=STRING       Change journal output mode (short, short-precise,\n"
+               "                                short-iso, short-full, short-monotonic, verbose,\n"
```
Comment (poettering) [inline src/journal/journalctl.c]: hmm, seems short-unix is also missing here, still...

**Comment by poettering** [comment]
Looks great, but I figure we should add the missing short-unix too here, as we are at it...

**Comment by keszybz** [comment]
Fixed. One more man page tweak added.

**Comment by poettering** [comment]
lgtm

**Pull request merged**